docker/cortx-deploy/build.sh

```diff
@@ -90,6 +90,10 @@ fi
 
 CREATED_DATE=$(date -u +'%Y-%m-%d %H:%M:%S%:z')
 
+
+if [ "$ENVIRONMENT" == "opensource-ci" ]; then
+        sed -i "/^[[:space:]].*TAG/a\    extra_hosts:\n      - $HOSTNAME: $(hostname -I | cut -d' ' -f1)" docker-compose.yml
+fi
```

Comment:
Please add that in steps instead of code. This solution will break when multiple images are added in the next sprint. We need to improve sed command to support multiple images. 
I feel it's better to update - https://github.com/Seagate/cortx/blob/main/doc/community-build/docker/cortx-all/README.md